diff --git a/node_modules/prompts/lib/elements/autocomplete.js b/node_modules/prompts/lib/elements/autocomplete.js
index b492414..31e29e8 100644
--- a/node_modules/prompts/lib/elements/autocomplete.js
+++ b/node_modules/prompts/lib/elements/autocomplete.js
@@ -152,14 +152,20 @@ class AutocompletePrompt extends Prompt {
   }
 
   up() {
-    if (this.select <= 0) return this.bell();
-    this.moveSelect(this.select - 1);
+    if (this.select === 0) {
+      this.moveSelect(this.suggestions.length - 1);
+    } else {
+      this.moveSelect(this.select - 1);
+    }
     this.render();
   }
 
   down() {
-    if (this.select >= this.suggestions.length - 1) return this.bell();
-    this.moveSelect(this.select + 1);
+    if (this.select === this.suggestions.length - 1) {
+      this.moveSelect(0);
+    } else {
+      this.moveSelect(this.select + 1);
+    }
     this.render();
   }
 
